sapi/fpm/tests/status-inherited-settings.phpt

```diff
@@ -0,0 +1,75 @@
+--TEST--
+FPM: Status pool is inheriting FPM setting overrides test
+--SKIPIF--
+<?php include "skipif.inc"; ?>
+--INI--
+expose_php=Off
```

Comment:
This will just confuse tester and has no meaning for fpm binary. It would be better remove it.